src/salmon/quant/config.vsh.yaml

```diff
@@ -0,0 +1,582 @@
+references: 
+  doi: "doi:10.1038/nmeth.4197"
+license: GPL-3.0 
+requirements:
+  commands: [ salmon ]
+
+argument_groups:
+  - name: Common input options
+    arguments:
+      - name: --lib_type
+        alternatives: ["-l"]
+        type: string
+        description: |
+          Format string describing the library.
+        required: false
+        example: 'A'
+        choices: ['A', 'U,', 'SF', 'SR', 'IU', 'IS', 'ISF', 'ISR', 'OU', 'OS', 'OSF', 'OSR', 'MU', 'MS', 'MSF', 'MSR']
+  
+  - name: Mapping input options
+    arguments:
+      - name: --index
+        alternatives: ["-i"]
+        type: file
+        description: |
+          Salmon index.
+        required: true
+        example: transcriptome_index
+      - name: --unmated_reads
+        alternatives: ["-r"]
+        type: file
+        description: |
+          List of files containing unmated reads of (e.g. single-end reads).
+        required: false
+        multiple: true
+        multiple_sep: " "
+        example: sample.fq.gz
+      - name: --mates1
+        alternatives: ["-m1"]
+        type: file
+        description: |
+          File containing the #1 mates.
+        required: false
+        multiple: true
+        multiple_sep: " "
+        example: sample_1.fq.gz
+      - name: --mates2
+        alternatives: ["-m2"]
+        type: file
+        description: |
+          File containing the #2 mates.
+        required: false
+        multiple: true
+        multiple_sep: " "
+        example: sample_2.fq.gz
+      
+  - name: Alignment input options
+    arguments:
+      - name: --discard_orphans
+        type: boolean_true
+        description: |
+          Discard orphan alignments in the input [for alignment-based mode only]. If this flag is passed, then only paired alignments will be considered toward quantification estimates. The default behavior is to consider orphan alignments if no valid paired mappings exist.
+      - name: --alignment
+        alternatives: ["-a"]
+        type: file
+        description: |
+          Input alignment (BAM) file(s).
+        required: false
+        multiple: true
+        multiple_sep: " "
+        example: sample.fq.gz
+      - name: --eqclasses
+        alternatives: ["-e"]
+        type: file
+        description: |
+          input salmon weighted equivalence class file.
+        required: false
+      - name: --targets
+        alternatives: ["-t"]
+        type: file
+        description: |
+          FASTA format file containing target transcripts.
+        required: false
+        example: transcripts.fasta
+      - name: --ont
+        type: boolean_true
+        description: |
+          Use alignment model for Oxford Nanopore long reads
+
+  - name: Output
+    arguments:
+      - name: --output
+        alternatives: ["-o"]
+        type: file
+        direction: output
+        description: |
+          Output quantification directory.
+        required: true
+        example: quant_output
```

Comment:
Can you split up this output directory? For example, the star_align_reads component looks at certain files it expects and will copy them to the desired output path : https://github.com/viash-hub/biobase/pull/22/files#diff-a13e23ef40c5889bceacefe5ae19749cb879ce3a2bab40b6145900e201af6d5bR54-R62 . If we output a single directory, it makes it harder for nextflow workflows to find out which files are available or not.

It's fine to also allow outputting the whole directory, as long as it's not required to do it this way.

Reply:
I've changed it to split the main output, i.e. the quantification file. So the output now consists of the output directory and a separate quantification file.